### DISCUSSION TITLE
Feature Request: FirebaseUI Support

### DISCUSSION THREAD

**Discussion by kroikie** [firebase/flutterfire#5549] [Feature request] [answered]
I really like FirebaseUI. It is very convenient and helps me focus on my application rather than a tedious authentication flow implementation. Can you make a FirebaseUI plugin for Flutter? Or implement it in dart?

**Comment by kroikie** [comment]
@clarktank

The issue at https://github.com/flutter/flutter/issues/18625 has been closed and moved here. Future collaboration on this issue will be done here.

**Comment by ciriousjoker** [comment]
Any news on this?

**Comment by Purus** [comment]
Any official package on this would be much reliable as other packages are incomplete.

**Comment by Salakar** [comment]
cc @Ehesp @rrousselGit potentially relevant for upcoming UI work.

**Comment by Rart3001** [comment]
Any update?

**Comment by Salakar** [comment, marked as answer]
This is now in progress - a UI roadmap issue detailing everything will be up in the next few days - keep an eye out for it.

**Comment by Purus** [comment]
In case, if someone is looking for details, please refer https://github.com/FirebaseExtended/flutterfire/discussions/6978.
